components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -146,10 +162,15 @@ void RRTMGPRadiation::run_impl (const Real dt) {
         rei(i+1,k+1)   = d_rei(i,k);
         p_lev(i+1,k+1) = d_pint(i,k);
         t_lev(i+1,k+1) = d_tint(i,k);
-
-        Kokkos::parallel_for(Kokkos::ThreadVectorRange(team, m_ngas), [&] (const int& g) {
-          gas_vmr(i+1,k+1,g+1) = d_gas_vmr(i,k,g);
-        });
+        // gas concentrations
+        gas_vmr(i+1,k+1,1) = d_qv(i,k);
```

Comment:
Another comment on this: RRTMGP expects these to be specified as volume mixing ratios, so these will need to be converted to conform with our policy of only supporting wet mixing ratios.